doc/source/building/faq.rst

```diff
@@ -23,7 +23,7 @@ command ``source scipy-dev/bin/activate``, and ``deactivate`` to exit from the
 virtual environment and back to your previous shell.  With scipy-dev
 activated, install first Scipy's dependencies::
 
-    $ pip install NumPy pytest Cython
+    $ pip install NumPy pytest Cython pybind11
```

Comment:
Looks good to me 👍. Just a minor point, maybe write all of them as either lowercase (prefered) or camel case?